### ISSUE TITLE
How to get 2:1 (1.9:1) aspect ratio??

### ISSUE THREAD

**Issue by enaveso** [cdgriffith/FastFlix#290]
Hi friends!
As a regular user of FastFlix I would like to know if there is any trick or way to get the 2:1 (1.9:1 IMAX) aspect ratio always. I always remove black bars using auto mode, but only sometimes do I get true 2:1 and the picture completely fills the TV screen.

**Comment by cdgriffith** [comment]
No way of automating that I can think of. Would have to manually calculate how much of the video you'd have to crop to achieve that ratio. However might be helpful for me to add a "ratio" indicator? To see if the output will be as expected.

**Comment by enaveso** [comment]
YES! a ratio indicator will be GREAT, so we can see the 2.48:1, 1.9:1, 1.78:1, 2:1 and so on... as Kodi shows...
